src/game/mod.rs

```diff
@@ -0,0 +1,227 @@
+use std::collections::HashMap;
+
+use crate::conways_error::ConwaysError;
+use crate::cell::Cell;
+
+pub struct Game {
+    live_cells: Vec<Cell>,
+}
+
+impl Game {
+    pub fn new_with_seed(seed: Vec<(i32,i32)>) -> Self {
```

Comment:
Maybe we could directly receive a `Vec<Cell>` instead of the tuple

Reply:
I thought about that when I created the method, but I was thinking that Cell is an abstraction that only the Game knows about, because maybe for an external user it is easier to use with just the tuples. What do you think about that?

Reply:
That's true, also the writing of the tests is easier with just a tuple than doing a `Cell::new()`, stay with the tuples

Reply:
Ignore my previous comment, it's been brought to my attention that it's a blatant rule to the orphan rule